src/org/thoughtcrime/securesms/loki/redesign/activities/SettingsActivity.kt

```diff
@@ -94,6 +95,8 @@ class SettingsActivity : PassphraseRequiredActionBarActivity() {
         linkedDevicesButton.setOnClickListener { showLinkedDevices() }
         seedButton.setOnClickListener { showSeed() }
         clearAllDataButton.setOnClickListener { clearAllData() }
+
+        versionText.text = String.format(getString(R.string.version_s), BuildConfig.VERSION_NAME)
```

Comment:
This is minor, but the empty line here breaks the code convention the rest of this file maintains